daml-script/daml/Daml/Script.daml

```diff
@@ -241,6 +243,9 @@ instance HasSubmit Script Commands where
   submitMustFail p cmds = lift $ Free (fmap pure $ Submit $ SubmitCmd p (fail <$> cmds) (const ()))
     where fail _ = error "Expected submit to fail but it succeeded"
 
+  submitMustFailMsg p cmds = lift $ Free (fmap pure $ Submit $ SubmitCmd p (fail <$> cmds) (const "help me"))
```

Comment:
I am a little stuck on the Daml Script implementation. 

1. The goal is to feed in the error message to where the "help me" placeholder is.

2. Are feature flags needed `#ifdef DAML_ANY_TYPE`? If so, I had issues when enabling the `CPP language pragma` relating to the config in `daml-opts/Options.hs`.